src/modules/avformat/filter_swresample.c

```diff
@@ -255,7 +259,7 @@ static int filter_get_audio( mlt_frame frame, void **buffer, mlt_audio_format *f
 		audio_format_planes( in_format, in_samples, in_channels, *buffer, pdata->in_buffers );
 		audio_format_planes( out_format, alloc_samples, out_channels, out_buffer, pdata->out_buffers );
 
-		int out_samples = swr_convert( pdata->ctx, pdata->out_buffers, alloc_samples, (const uint8_t**)pdata->in_buffers, in_samples );
```

Comment:
BTW this C99 feature of intermingling variable declarations is permitted without a compiler flag and used throughout the code, but I am not going to reject the pull request, of course.